tools/inflections-xml2csv/converter.js

```diff
@@ -1644,8 +1644,9 @@ try {
         let j = xmlToJSON.parseString(d)
         json['infl-data'] = json['infl-data'].concat(j['infl-data'])
       }
-      writeData(posCfg.suffixes.get(json), posCfg.suffixes.outputPath)
-      writeData(posCfg.footnotes.get(json), posCfg.footnotes.outputPath)
+      // Suffixes were edited manually. Running this conversion would erase those changes
```

Comment:
Ah, I didn't realize we were still running the conversions. Maybe we should stop that for all tables, and make the csv files the official data files now?

Reply:
We're usually running this script once for each new piece of data. But I will probably comment out all conversion routines for safety.